app/views/includes/foot.jade

```diff
@@ -3,16 +3,16 @@ script.
       (function(d,t){var g=d.createElement(t),s=d.getElementsByTagName(t)[0];
       g.src='//www.google-analytics.com/ga.js';
       s.parentNode.insertBefore(g,s)}(document,'script'));
-script(type='text/javascript', src='/lib/jquery/jquery.js')
+script(type='text/javascript', src='/lib/jquery/jquery.min.js')
 script(type='text/javascript', src='/lib/underscore/underscore-min.js')
 
 //Bootstrap
-script(type='text/javascript', src='/lib/bootstrap/js/bootstrap.js')
+script(type='text/javascript', src='/lib/bootstrap/dist/js/bootstrap.min.js')
 
 //AngularJS
-script(type='text/javascript', src='http://code.angularjs.org/1.1.5/angular.js')
-script(type='text/javascript', src='http://code.angularjs.org/1.1.5/angular-resource.js')
-script(type='text/javascript', src='http://code.angularjs.org/1.1.5/angular-cookies.js')
+script(type='text/javascript', src='https://code.angularjs.org/1.1.5/angular.js')
+script(type='text/javascript', src='https://code.angularjs.org/1.1.5/angular-resource.js')
+script(type='text/javascript', src='https://code.angularjs.org/1.1.5/angular-cookies.js')
```

Comment:
You should be able to use `bower install angular --save` to save angular locally. And reference it like this --> `script(type='text/javascript', src='/lib/angular/angular.min.js')`. The same thing for `angular-cookies` and `angular-resources`